### DISCUSSION TITLE
How to add SearXNG to microsoft edge?

### DISCUSSION THREAD

**Discussion by nandgator** [searxng/searxng#1866] [Q&A]
![image](https://user-images.githubusercontent.com/38415384/196658499-d4c9dcfb-6602-411c-9c61-81a3e06c21c2.png)

> I'd like to keep it POST, is it possible?

**Comment by return42** [comment]
Don't add it by hand .. visit http://127.0.0.1:8888/info/de/about and see "How do I set it as the default search engine?".

**Reply by nandgator** [reply]
![image](https://user-images.githubusercontent.com/38415384/196756282-1fa5f987-6fa0-4498-b78e-270a7dbc8e03.png)

For edge this section ☝🏻 leads to: https://support.microsoft.com/en-us/microsoft-edge/change-your-default-search-engine-f863c519-5994-a8ed-6859-00fbc123b782

![image](https://user-images.githubusercontent.com/38415384/196756452-97840b57-51ae-4e3d-87f4-0bfcd17f269d.png)

Which is of no use since, `SearXNG` does not show up at all (even after a search is made):

![image](https://user-images.githubusercontent.com/38415384/196756664-e6fb03eb-3ab9-4c5f-89e6-05028985197d.png)

**Reply by return42** [reply]
Edge is Chrome https://www.chromium.org/tab-to-search/ .. and it works:

![grafik](https://user-images.githubusercontent.com/554536/196869303-21ce8304-9ce1-4aaa-a1d8-7cb54696813b.png)

**Comment by nandgator** [comment]
**Update:** This works 👇🏻 

![image](https://user-images.githubusercontent.com/38415384/196758916-da48edcd-239f-49b4-93af-d440caa7b18d.png)

> But now it's not longer a POST request

**Reply by return42** [reply]
> **Update:** This works 👇🏻

When adding manually, you miss autocompletion --> https://github.com/searxng/searxng/issues/431

**Reply by p6002** [reply]
So how to do it correctly?

**Reply by return42** [reply]
![grafik](https://user-images.githubusercontent.com/554536/217866782-b12dfd1c-6941-4979-8e14-f0113bfdaa31.png)

But please note, some browser do not support "OpenSearch" .. for the most browsers of the Chrome and Firefox derivate it should work .. but not those Firefox & Chrome browsers you will find on a (mobile) Android device .. most of them do not support OpenSearch ..  thats why I opened a discussion --> https://github.com/searxng/searxng/issues/431 


Update forgot to say: [if you have problems adding a SearXNG instance](https://github.com/searxng/searxng/discussions/1866#discussioncomment-3921388) (you downvoted my answer, so I guess you have problems with) .. then please post the URL of the SearXNG instance and what browser you are using .. I will have a look and we can solve the issue.
